src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write code here
+const listItems = document.querySelectorAll('#employeeList > li');
+
+const getSalary = (item) => {
+  const salaryString = item.getAttribute('data-salary').replace(/[$,]/g, '');
+
+  return parseFloat(salaryString);
+};
+
+const sortList = (list) => {
+  return Array.from(list).sort((a, b) => getSalary(b) - getSalary(a));
+};
+
+const sortedList = sortList(listItems);
+
+const employeeList = document.getElementById('employeeList');
+
+employeeList.innerHTML = '';
+sortedList.forEach((item) => employeeList.appendChild(item));
+
+// const employees = getEmployees(sortedList);
+// console.log(employees);
```

Comment:
in demo link items not sorted.
- append to list in sortList funcion
- write function, which returns an array of objects, where objects are employees
- call two functions

Check description in readme.md for more details.